cli/src/plugins/javadoc/index.ts

```diff
@@ -182,6 +183,57 @@ function makeSuperclassList(project: Project, doc: ParsedClassDoc) {
   ];
 }
 
+function makeInterfaceMethodList(project: Project, doc: ParsedClassDoc, interfaceTypes: Type[], inheritedMethods: {[k: string]: string[]}) {
+  if (interfaceTypes.length === 0) {
+    return [];
+  }
+  return [
+    md.heading(2, md.text("Methods Inherited from Interfaces")),
+    md.root(interfaceTypes.map((interfaceType) => {
+      return [
+        md.heading(3, md.text(interfaceType.qualifiedTypeName)),
+        md.list(
+          "unordered",
+          inheritedMethods[interfaceType.qualifiedTypeName].map((method) => {
+            return md.listItem(project.linkToEntity([interfaceType.qualifiedTypeName, method].join('.'), method))
+          }))
+      ]
+    }).flat(1))
+  ]
+}
+
+function makeSuperclassMethodList(project: Project, doc: ParsedClassDoc, superclasses: Type[], inheritedMethods: {[k: string]: string[]}) {
+  if (superclasses.length === 0) {
+    return [];
+  }
+  return [
+    md.heading(2, md.text("Methods Inherited from Superclasses")),
+    md.root(superclasses.map((superclass) => {
+      return [
+        md.heading(3, md.text(superclass.qualifiedTypeName)),
+        md.list(
```

Comment:
Yeah I don't know about the list either. What's wrong with comma-separated list? E.g.:

- Class or interface name: method, method, method

Reply:
Is it overkill to have this separated by interfaces/classes? If we wanted to do this we should have done so in the doclet.

Reply:
I'm trying a list of lists in this next iteration, with links to the overarching class in each method just like the javadoc provides right now. If we don't like that, we can possibly even switch to a table, or at least a comma separated list of methods after a single class link.